pom.xml

```diff
@@ -121,7 +121,7 @@ flexible messaging model and an intuitive client API.</description>
     <testRealAWS>false</testRealAWS>
     <testRetryCount>1</testRetryCount>
 
-    <bookkeeper.version>4.7.0</bookkeeper.version>
+    <bookkeeper.version>4.7.1</bookkeeper.version>
```

Comment:
Where is the equivalent changes in javainstancemain and python instance main

Reply:
not sure what do you mean here? but all the versions are managed at the root pom.xml